Src/Public/Invoke-AsBuiltReport.Fortinet.Fortigate.ps1

```diff
@@ -29,13 +29,26 @@ function Invoke-AsBuiltReport.Fortinet.Fortigate {
     # Used to set values to TitleCase where required
     $TextInfo = (Get-Culture).TextInfo
 
-	# Update/rename the $System variable and build out your code within the ForEach loop. The ForEach loop enables AsBuiltReport to generate an as built configuration against multiple defined targets.
-	
+    # Update/rename the $System variable and build out your code within the ForEach loop. The ForEach loop enables AsBuiltReport to generate an as built configuration against multiple defined targets.
+
     #region foreach loop
     foreach ($System in $Target) {
-		
-		
-		
-	}
-	#endregion foreach loop
+
+        #Connection to Fortigate (TODO: Add Paremeter for Certificate Check and Port)
+        Connect-FGT -Server $System -Credential $Credential -SkipCertificateCheck | Out-Null #-Port $Options.ServerPort
+
+        #Get Model
+        $Model = (Get-FGTMonitorSystemFirmware).current.'platform-id'
+        Write-PScriboMessage "Connect to $System : $Model ($($DefaultFGTConnection.serial)) "
+
+        Section -Style Heading1 'Implementation Report' {
```

Comment:
Use the system name as the top level heading, this works better when running against multiple systems